### PR TITLE
feat(TJS): handle `examples` annotation

### LINKED ISSUE #259: feature request: support `@examples` annotation -> json schema `examples`
It seems that `@description` and `@default` keywords are supported but not `@examples` (or @title)

https://json-schema.org/understanding-json-schema/reference/generic.html

### PR DESCRIPTION
**Pull request by evaneus**
fixes #259

Please:
- [X] Do not include the compiled `.js`, `.js.map`, or `.d.ts` in your pull request as it makes it harder to merge your changes. 
- [X] Make your pull request atomic, fixing one issue at a time unless there are many relevant issues that cannot be decoupled.
- [x] Provide a test case & update the documentation in the `Readme.md`

### CODE REVIEW

File: typescript-json-schema.ts [hunk 1]
```diff
@@ -268,6 +269,7 @@ const validationKeywords = {
     // anyOf: true,                    // 6.27.
     // oneOf: true,                    // 6.28.
     // not: true,                      // 6.29.
+    examples:true,                     // Draft 6 (draft-handrews-json-schema-validation-01)
```
Comment (domoritz) [inline typescript-json-schema.ts]: Add space

File: typescript-json-schema.ts [hunk 2]
```diff
@@ -70,6 +70,7 @@ export type Definition = {
     $schema?: string,
     $id?: string,
     description?: string,
+    examples?: string[],
```
Comment (domoritz) [inline typescript-json-schema.ts]: Does an example always have to be a string?

Comment (evaneus) [reply typescript-json-schema.ts]: Doesn't seem to make any difference. Can resubmit with `any[]`

> There are no restrictions placed on the values within the array...

 > This keyword can be used to provide sample JSON values associated
   with a particular schema, for the purpose of illustrating usage.

File: typescript-json-schema.ts [hunk 3]
```diff
@@ -269,7 +269,7 @@ const validationKeywords = {
     // anyOf: true,                    // 6.27.
     // oneOf: true,                    // 6.28.
     // not: true,                      // 6.29.
-    examples:true,                     // Draft 6 (draft-handrews-json-schema-validation-01)
+    examples: true,                     // Draft 6 (draft-handrews-json-schema-validation-01)
```
Comment (domoritz) [inline typescript-json-schema.ts]: Now the comments are not aligned anymore

**Comment by domoritz** [comment]
Thanks!

**Comment by domoritz** [comment]
Could you send a follow up PR where you add a list of supported annotation keywords to the readme?

**Pull request merged**